src/docs/product/integrations/aws-lambda/index.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: AWS Lambda (Node)
+sidebar_order: 1
+description: "Learn more about Sentry's AWS Lambda integration and how you can automatically instrument your Node Lambda functions with Sentry without changing your code."
+---
+
+<Alert title="Note" level="info">
+
+Looking for instructions to set up the Node serverless SDK manually? [Check out these docs instead.](/platforms/node/guides/aws-lambda/)
+
+</Alert>
+
+Connect Sentry to your AWS account to automatically add Sentry error monitoring and transactions to your Node Lambda functions.
+
+
+## Install
+
+1. Navigate to **Settings > Integrations > AWS Lambda**.
+
+   ![](directory.png)
+
+1. Press **Add Installation** which will open the installation modal.
+
+   ![](integration_detail.png)
+
+1. Select a project to use for your Lambda functions. Note that this project cannot be changed later. If you only have a single project, this step is skipped.
```

Comment:
Can we better explain what we mean by "this project cannot be changed later" - does that mean if you select the wrong project, you can't just go back, you'd need to uninstall, then start the process over with a new project?
```suggestion
1. Select the project for your Lambda integration. **Note:** This project cannot be changed later. If you only have a single project, you'll skip this step.
```